org.getmonero.Monero.xml

```diff
@@ -2,12 +2,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!-- Copyright 2018-2021 Flathub maintainers -->
 <component type="desktop">
-  <id>org.getmonero.monero.desktop</id>
-  <launchable type="desktop-id">org.getmonero.monero-wallet.desktop</launchable>
```

Comment:
org.getmoneor.MoneroWallet would also be acceptable.

This file, the appstream metadata, should be named `org.getmonero.Monero.metainfo.xml`

Reply:
Hi, yes I made that change in my other branch, which should be visible in the new PR I recently created.